### DISCUSSION TITLE
Trying to import a database with bank information on Odoo 17 and can't get past the error message.

### DISCUSSION THREAD

**Discussion by SirJump** [OCA/odoo-community.org#177] [Q&A]
Hey, trying to import a database with bank information on Odoo 17 and can't get past the error message. Does anyone have any idea how to change the data so Odoo accepts it.

thank you in advance.
![Unbenannt](https://github.com/user-attachments/assets/1f88bc50-f08c-4f9a-9dbc-2366a3ede442)

**Comment by vdewulf** [comment]
Hello @SirJump ,
Not sure what data you want to import: the list of banks or a list of contacts together with their bank account data?
The error indicates a missing mandatory name on row 0, but row 0 should be the column titles. You should click on "The first row contains the label of the column" option in the import screen.
Hope this might help?!

**Reply by SirJump** [reply]
Hey, thank you for your reply.
I am trying to add contacts that already have Bank information in my CSV sheet.
But somehow odoo does not know what to do with the information even thought I have given it the correct id/connection of where this information needs to be placed.

> The error indicates a missing mandatory name on row 0, but row 0 should be the column titles. You should click on "The first row contains the label of the column" option in the import screen.

I did that, and odoo does acept the date if I tell it to ignore the bank data but one ll try to add it, i get asked for "contacts require name" error

thak you in advace for your time
